package.json

```diff
@@ -20,7 +20,7 @@
     "AWS"
   ],
   "author": "Wyatt Pearsall <wyatt.pearsall@cfpb.gov>",
-  "license": "CC0",
+  "license": "MIT",
```

Comment:
Why the switch to "MIT"?  Shouldn't this be "Public Domain"?  Browsing around some of our other repos, looks like we generally have something like:

```json
"licenses": [
    {
      "type": "Public Domain",
      "url": "http://github.com/cfpb/grasshopper-retriever/blob/master/TERMS.md"
    }
  ]

```